src/main/java/uk/ac/cam/cl/dtg/segue/dao/users/PgUsers.java

```diff
@@ -727,8 +727,8 @@ private RegisteredUser createUser(final RegisteredUser userToCreate) throws Segu
 
         String query = "INSERT INTO users(family_name, given_name, email, role, date_of_birth, gender," +
                 " registration_date, school_id, school_other, last_updated, email_verification_status, last_seen," +
-                " email_verification_token, email_to_verify, session_token, registered_contexts, registered_contexts_last_confirmed)" +
-                " VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);";
+                " email_verification_token, email_to_verify, teacher_pending, session_token, registered_contexts, registered_contexts_last_confirmed)" +
```

Comment:
Just to double-check, did you create this branch from the teacher_pending one rather than main?
Probably shouldn't be an issue, just make sure we order/coordinate the merges correctly

Reply:
Should be fine, had to merge the teacher_pending branch into the branch because needed the changes for registration.